### PR TITLE
add KeystoneJS

### PR DESCRIPTION
**Pull request by yuche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Grav](http://getgrav.org/) -  Modern, Crazy Fast, Ridiculously Easy and Amazingly Powerful Flat-File CMS  ([Source code](https://github.com/getgrav/grav)) `MIT` `PHP`
   * [GetSimple CMS](http://get-simple.info/) - The Simplest Content Management System. Ever. `GPLv3`
   * [Joomla!](http://www.joomla.fr/) - Advanced Content Management System (CMS) `GPL`
+  * [KeystoneJS](http://keystonejs.com/) - Node.js CMS & Web Application Platform ([Demo](http://demo.keystonejs.com/), [Source Code](https://github.com/keystonejs/keystone)) `MIT`
```
Comment (nodiscc) [inline README.md]: can you move Nodejs at line end between backticks `` ` ``? thank you for the patch

Comment (yuche) [inline README.md]: done.

**Comment by Kickball** [comment]
@nodiscc Anything wrong here? As far as I'm concerned we're ready to merge this.

**Comment by nodiscc** [comment]
it was missing nodejs but yuche fixed it (_nodiscc commented on an outdated diff_)
good to merge!

**Pull request merged**